CIP-xxxx/README.md

```diff
@@ -0,0 +1,127 @@
+
+All involved parties (SPOs, DAP developers, etc) need to agree on SRV prefixes used by various DAPs and thus a public registry governed by a public CIP process is required to foster DAPs that co-exist with the **Cardano** network, like **Mithril** (see [CIP#137]).
+
+Note that the notion of a DAP is specific in this context: these are applications, which construct their own network and relay on ledger peers published on the block-chain.   This include node implemtations, or 2nd-layer solutions like Mithril or Hydra.
+
+### Problem
+
+Bootstraping networks requires sharing information about available services.
+
+Historically, this was done by sharing IP or DNS names and PORT numbers, but this approach has its limitations.
+
+SRV records were designed to make deployment independent of hard-coded service end-points (see [RFC#2782]).
+
+Thus they can be instrumental in `Cardano`, since relays are registered on the blockchain through the registration certificate.
+
+By using SRV records in the registration certificate (which is supported by the `cardano-ledger`, but not by `cardano-node`), we wish to solve this problem not just for `Cardano` node implementation, but also for any DAP that requires constructing its own network.
+
+SRV provide a mechanism for exposing DAPs co-deployed with a node, like **mithril** or **hydra**.
+
+Making such services discoverable is one of the key features addressed by this CIP.
+
+In this CIP, we propose prefixes for both **Cardano** and **Mithril**; in the future, other services can be registered through a CIP process, thus starting a registry of prefixes used by the **Cardano** ecosystem.
+
+SPOs who deploy services need to configure their system according to the registry, e.g. SPO's cardano relays node MUST be available at `_cardano._tcp.<SPO_DOMAIN_NAME>`, as other nodes on the system will be looking at this address.
+
+DAP developers SHOULD submit proposals to the SRV Prefix Registry, so SPOs, who deploy them, can have an authoritative information how to do it.
+
+## Specification
+
+### SRV Prefix Registry
+
+| **service** | **scope**     | **SRV prefix**                       | **Status** |
+|:------------|:--------------| ------------------------------------:|:-----------|
+| _cardano_   |               |                      `_cardano._tcp` | Active     |
+| _mithril_   | _dmq_         |       `_.dmq._mithril._cardano._tcp` | Active     |
+| _mithril_   | _aggregator_  | `_aggregator._mithril._cardano._tcp` | Active     |
+
+### SRV Prefix Semantics
+
+When a **cardano** node implementation reads an SRV record from a ledger, it must add the _cardano_ prefix from the table above before making a DNS lookup, e.g. it should do a DNS query for `_cardano._tcp.<srv-record-from-ledger>`.
+
+This design allows DAPs to use SRV records registered in the ledger for different purposes, e.g. a **mithril** node can use them to learn about end-points of its network.
```

Comment:
Does this mean that someone could use a stake pool registration certificate to ONLY register a mithril node? and not a Cardano node